tests/VerifyHandlerTest.php

```diff
@@ -119,13 +121,21 @@ public function testVerify()
 
     public function testExceptionsOnStartMethodAreLogged()
     {
-        Environment::setEnv('SS_MFA_SECRET_KEY', null);
+        putenv('SS_MFA_SECRET_KEY=');
 
-        /** @var LoggerInterface|PHPUnit_Framework_MockObject_MockObject $logger */
-        $logger = $this->createMock(LoggerInterface::class);
-        $logger->expects($this->once())->method('debug');
+        /** @var Zend_Log_Writer_Null|PHPUnit_Framework_MockObject_MockObject $logger */
+        $logger = $this->createMock(Zend_Log_Writer_Null::class);
+        $logger->expects($this->once())->method('write');
+
+        $originalWriters = SS_Log::get_writers();
```

Comment:
Do we need this now? We should just use `SS_Log::log`...

Reply:
Yes, this test is specifically testing the logging, the mocking code does not exist in order to prevent side effects from global state as they have been employed (and subsequently removed) elsewhere.